kolibri/plugins/learn/assets/src/views/content-card-group-grid/index.vue

```diff
@@ -91,6 +91,9 @@
       channelFilterSelection: '',
     }),
     computed: {
+      channelKind() {
```

Comment:
I don't think this method is necessary anymore.

Reply:
Fixed.